### ISSUE TITLE
Some questions about the ‘bce_iou_loss’ function

### ISSUE THREAD

**Issue by Liqq1** [plemeri/UACANet#6]
```
def bce_iou_loss(pred, mask):
    weight = 1 + 5 * torch.abs(F.avg_pool2d(mask, kernel_size=31, stride=1, padding=15) - mask)
  
    bce = F.binary_cross_entropy_with_logits(pred, mask, reduction='none')
   
    pred = torch.sigmoid(pred)
    inter = pred * mask
    union = pred + mask
    iou = 1 - (inter + 1) / (union - inter + 1)

    weighted_bce = (weight * bce).sum(dim=(2, 3)) / weight.sum(dim=(2, 3))
    weighted_iou = (weight * iou).sum(dim=(2, 3)) / weight.sum(dim=(2, 3))

    return (weighted_bce + weighted_iou).mean()
```
![question](https://user-images.githubusercontent.com/54054113/150813608-083c3d92-c629-4bd0-babb-7115c74cf1bd.jpg)

I am looking forward to your reply！

**Comment by plemeri** [comment]
Sorry for the late reply.
As we used [PraNet](https://arxiv.org/abs/2006.11392) as our baseline, we thought the loss function was also from them. In Fact, the authors of PraNet referred [F3Net](https://arxiv.org/abs/1911.11445) which proposed the loss function that we used, and we should have referred them too. 

Long story short, sorry that we mislead some information, please refer to F3Net for more information.

**Comment by Liqq1** [comment]
Thanks for your reply, I will refer to F3Net for more information～
